main.go

```diff
@@ -119,31 +119,17 @@ func init() { //nolint:gochecknoinits
 }
 
 func initComponents(_ context.Context, p cluster.Platform) error {
-	var multiErr *multierror.Error
-
-	if err := dashboardctrl.Init(p); err != nil {
-		multiErr = multierror.Append(multiErr, err)
-	}
-	if err := rayctrl.Init(p); err != nil {
-		multiErr = multierror.Append(multiErr, err)
-	}
-	if err := modelregistryctrl.Init(p); err != nil {
-		return err
-	}
-	if err := trainingoperatorctrl.Init(p); err != nil {
-		return err
-	}
-	if err := trustyaictrl.Init(p); err != nil {
-		return err
-	}
-	if err := datasciencepipelinesctrl.Init(p); err != nil {
-		multiErr = multierror.Append(multiErr, err)
-	}
-
-	if err := kueuectrl.Init(p); err != nil {
-		multiErr = multierror.Append(multiErr, err)
-	}
-	return multiErr.ErrorOrNil()
+	dashboardctrl.Init()
+	rayctrl.Init()
+	modelregistryctrl.Init()
+	trainingoperatorctrl.Init()
+	trustyaictrl.Init()
+	kueuectrl.Init()
+	datasciencepipelinesctrl.Init()
```

Comment:
no objection to use an `init()` function to get rid of those "component.Init()"

Reply:
> no objection to use an `init()` function to get rid of those "component.Init()"

Updated commit message, but forgot about PR description :)

* Use explicit Init() functionality to register components. Automatic init() does not work since it requires package to be imported to be executed (without explicit call there is no need to import the packages)*

Reply:
Updated with `_` imports based on side conversation.